### ISSUE TITLE
Need to export path to start ucoind

### ISSUE THREAD

**Issue by Insoleet** [ucoin-io/ucoin#251]
Command `ucoind start` does not work.

I need to export path first : 
```
export PATH=$PATH:$UCOIN_DIR/node/bin/
ucoind start
```

Without it, it does not find node binary.

**Comment by c-geek** [comment]
What is your "bash" program?

**Comment by Insoleet** [comment]
bash.

Could it be because I installed ucoin via the install script when it was buggy, and updating it throught the install.sh never fixed it ?

Maybe I should just add manually the export to my .bashrc ?

**Comment by c-geek** [comment]
Yes that's possible. You can remove the 2 lines added by uCoin installer:

```
export UCOIN_DIR="/home/cgeek/.ucoin" 
133 [ -s "$UCOIN_DIR/ucoin.sh" ] && . "$UCOIN_DIR/ucoin.sh"  # This loads ucoin.sh
```

**Comment by c-geek** [comment]
And you should reinstall uCoin then.

**Comment by Insoleet** [comment]
It adds the lines to the .bashrc but I still don't have node binary in my PATH.

```
ucoin@vps55093:~$ ucoind logs
/usr/bin/env: node: No such file or directory
```

I think the script should append the export of the path to the .bashrc file.

**Comment by c-geek** [comment]
What `which ucoind` returns?

You probably have an old uCoin version installed in `/usr/bin`. In such case, you can remove it with:

```
$ sudo npm remove -g ucoin
```

Then ucoind command will really call the new ucoin version.

**Comment by Insoleet** [comment]
```
ucoin@vps55093:/root$ which ucoind
ucoin@vps55093:/root$ 
```

Note that nodejs is not installed in my environment.

**Comment by c-geek** [comment]
Good catch, the problem is PM2.

**Comment by c-geek** [comment]
I released [0.12.2](https://github.com/ucoin-io/ucoin/releases/tag/v0.12.2) that should fix the problem.

You should be able to update using [update procedure](https://github.com/ucoin-io/ucoin/wiki/Updating-uCoin).
